### PR TITLE
Serving specs

### PR DESCRIPTION
**Pull request by tayarani**
# Changes

- Part of https://github.com/knative/specs/issues/4
- Consists of 2 commits:
- https://github.com/knative/specs/pull/7/commits/69cf96d3db2337e423ab45d2f843ac269619f532 copies over files as-is from serving/docs, serving/docs/spec, and docs/serving/spec/
- https://github.com/knative/specs/pull/7/commits/31747e2e16379c4f00df693eef9489e10913224b with changes to fix broken links, and delete obsolete file.
- See report of link checking below.

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Comment by tayarani** [comment]
FILE: /tmp/README.md
[✓] https://github.com/knative/serving/tree/main/test/conformance
[✓] motivation.md
[✓] overview.md
[✓] knative-api-specification-1.0.md
[✓] runtime-contract.md

5 links checked.

FILE: /tmp/errors.md
[✓] https://github.com/knative/docs/blob/main/docs/serving/spec/knative-api-specification-1.0.md#error-signalling
[✓] https://github.com/knative/docs/blob/main/docs/serving/spec/knative-api-specification-1.0.md
[✓] https://github.com/knative/docs/tree/main/docs/serving/spec

3 links checked.

FILE: /tmp/knative-api-specification-1.0.md
[✓] https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/resource-management.md
[✓] https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
[✓] https://github.com/knative/serving/blob/main/docs/runtime-contract.md
[✓] https://tools.ietf.org/html/rfc2119
[✓] #resource-overview
[✓] https://grpc.io/
[✓] https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
[✓] https://tools.ietf.org/html/rfc7386
[✓] https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties
[✓] https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/principles.md#control-logic
[✓] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#ownerreference-v1-meta
[✓] https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#controlling-how-the-garbage-collector-deletes-dependents
[✓] #detailed-resources--v1
[✓] https://tools.ietf.org/html/rfc7230#section-5.4
[✓] https://tools.ietf.org/html/rfc7230#section-6.3
[✓] https://tools.ietf.org/html/rfc7540#section-5
[✓] https://tools.ietf.org/html/rfc7540#section-8.1
[✓] https://tools.ietf.org/html/rfc4592
[✓] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta
[✓] https://github.com/knative/pkg/blob/main/apis/duck/v1/addressable_types.go
[✓] mailto:dangerd@google.com
[✓] mailto:dug@us.ibm.com
[✓] mailto:evana@vmware.com

23 links checked.

FILE: /tmp/motivation.md
No hyperlinks found!

0 links checked.

FILE: /tmp/normative_examples.md
[✓] https://github.com/knative/client

1 links checked.

FILE: /tmp/overview.md
[✓] images/object_model.png

1 links checked.

FILE: /tmp/runtime-contract.md
[✓] https://github.com/opencontainers/runtime-spec/blob/master/spec.md
[✓] knative-api-specification-1.0.md
[✓] https://tools.ietf.org/html/rfc2119
[✓] https://github.com/opencontainers/runtime-spec/blob/v1.0.1/spec.md
[✓] https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md
[✓] https://github.com/knative/serving/issues/848
[✓] https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#kill
[✓] https://github.com/hyperhq/runv
[✓] https://katacontainers.io/
[✓] https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/runtime.md#operations
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/container_test.go
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/file_descriptor_test.go
[✓] https://github.com/knative/docs/issues/902
[✓] https://httpwg.org/specs/rfc7540.html#discover-http
[✓] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#containerport-v1-core
[✓] https://httpwg.org/specs/rfc7540.html#known-http
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/header_test.go
[✓] https://tools.ietf.org/html/rfc7230#section-5.4
[✓] https://tools.ietf.org/html/rfc7239
[✓] https://www.w3.org/2018/04/distributed-tracing-wg-charter.html
[✓] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#container-v1-core
[✓] https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-readiness-probes
[✓] https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-http-request
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/readiness_probe_test.go
[✓] #default-filesystems
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/filesystem_test.go
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/envvars_test.go
[✓] #inbound-network-connectivity
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/user_test.go
[✓] https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#default-devices
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/cgroup_test.go
[✓] https://github.com/knative/serving/blob/main/test/conformance/runtime/sysctl_test.go
[✓] https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt

33 links checked.

**Review by evankanderson — COMMENTED** [review]
Did you expect the `object_model.png` file to be empty?

Will review the other files in a bit (assuming they are direct copies of what's in `main`).

**Comment by tayarani** [comment]
@evankanderson the file is not empty. That looks like a UI bug in github. If you click on view file you can see the image. I also checked the branch locally and the file is 16kb, not empty.

Yes the files are copied over. All the changes I made are in https://github.com/knative/specs/commit/31747e2e16379c4f00df693eef9489e10913224b to simplify the review process.

File: specs/serving/normative_examples.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Sample API Usage
+
+Serving client documentation can now be found in the
+[Knative Client Repository](https://github.com/knative/client).
```
Comment (evankanderson) [inline specs/serving/normative_examples.md:4]: Again, fine with leaving this in, but not sure we need it at all.

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

A couple notes that can be fixed up in future PRs.

**Pull request merged**